pages/api/session.js

```diff
@@ -1,22 +1,30 @@
 import People from '@klaudsol/commons/models/People';
 import { withSession } from '@klaudsol/commons/lib/Session';
 import { defaultErrorHandler } from '@klaudsol/commons/lib/ErrorHandler';
+import { setCORSHeaders } from "@klaudsol/commons/lib/API";
 import { OK, UNPROCESSABLE_ENTITY } from '@klaudsol/commons/lib/HttpStatuses';
 import UnauthorizedError from '@klaudsol/commons/errors/UnauthorizedError';
 import Session from '@klaudsol/commons/models/Session';
 import { assertUserIsLoggedIn } from '@klaudsol/commons/lib/Permissions';
 import EntityType from '@/backend/models/core/EntityType';
+import { setCookie } from 'cookies-next';
+import { generateToken } from '@klaudsol/commons/lib/JWT';
 
 export default withSession(handler);
 
 
 async function handler(req, res) {
+  // i hate cors
+  setCORSHeaders({ response: res, url: process.env.FRONTEND_URL });
```

Comment:
Let's prefix this with KS_
```suggestion
  setCORSHeaders({ response: res, url: process.env.KS_FRONTEND_URL ?? process.env.FRONTEND_URL });
```